cpp/src/utilities/wrapper_types.hpp

```diff
@@ -157,66 +175,30 @@ template <typename T, gdf_dtype type_id>
 CUDA_HOST_DEVICE_CALLABLE
 wrapper<T,type_id> operator+(wrapper<T, type_id> const &lhs, wrapper<T, type_id> const &rhs)
 {
-  return wrapper<T, type_id>{lhs.value + rhs.value};
+  return wrapper<T, type_id>{ static_cast<T>(lhs.value + rhs.value) };
 }
 
 template <typename T, gdf_dtype type_id>
 CUDA_HOST_DEVICE_CALLABLE
 wrapper<T,type_id> operator-(wrapper<T,type_id> const& lhs, wrapper<T,type_id> const& rhs)
 {
-  return wrapper<T, type_id>{lhs.value - rhs.value};
+  return wrapper<T, type_id>{ static_cast<T>(lhs.value - rhs.value) };
 }
 
 template <typename T, gdf_dtype type_id>
 CUDA_HOST_DEVICE_CALLABLE
 wrapper<T,type_id> operator*(wrapper<T,type_id> const& lhs, wrapper<T,type_id> const& rhs)
 {
-  return wrapper<T, type_id>{lhs.value * rhs.value};
+  return wrapper<T, type_id>{ static_cast<T>(lhs.value * rhs.value) };
 }
 
 template <typename T, gdf_dtype type_id>
 CUDA_HOST_DEVICE_CALLABLE
 wrapper<T,type_id> operator/(wrapper<T,type_id> const& lhs, wrapper<T,type_id> const& rhs)
 {
-  return wrapper<T, type_id>{lhs.value / rhs.value};
```

Comment:
wrapper prefix and postfix operators were deleted.

Reply:
In more detail: they are problematic for bool8 since `--` is undefined for `bool` in C++. This lead to gymnastics for testing, which made us realize that the increment/decrement operators were added without a lot of thought and aren't used much. We decided to remove for now and address if it becomes a problem.